backend/typescript/services/implementations/schedulingService.ts

```diff
@@ -737,7 +737,7 @@ class SchedulingService implements ISchedulingService {
           cancellationEmail(donorMainLine, donor.firstName),
         );
         this.emailService.sendEmail(
-          "communityfridgekw@gmail.com",
+          getAdminEmail(),
```

Comment:
re: slack comment to change 
https://github.com/uwblueprint/community-fridge-kw/blob/d114da62de30c5c382a635ae04653d555ab3d95b/backend/typescript/utilities/emailUtils.ts#L58